components/math/multiplicationConnect/GameBoard.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import { getRndInteger } from '../../../pages/api/random';
+
+const createGrid = () => {
+    let arr = [];
+    do{
+        let gridNumber = getRndInteger(4, 25);
+        (gridNumber%2===0) ? (
```

Comment:
This can slow down your user experience greatly. If you keep getting odd numbers your user will be stuck in an infinite loop and it'll crash their browser.

Is there another way we can achieve this?

Reply:
We have a helper function called get random item from array. Why don't you create an array of all valid values and pick a random item from there?

Reply:
I couldn't think of a workaround but this is a good idea, thanks!